src/Message/DirectAuthorizeRequest.php

```diff
@@ -54,6 +54,8 @@ protected function getBaseAuthorizeData()
         $data['DeliveryPhone'] = $card->getShippingPhone();
         $data['CustomerEMail'] = $card->getEmail();
 
+        $data = array_merge($data, $this->getItemData());
```

Comment:
Okay, so a check for an empty basket is needed, so the field can be left out completely. You don't need to create a new PR for that - just push your changes to your PR branch and it should update the PR automatically.